backend/core/src/entity/unit.entity.ts

```diff
@@ -149,6 +149,11 @@ class Unit extends BaseEntity {
     onUpdate: "CASCADE",
   })
   listing: Listing
+
+  @Column({ type: "boolean", nullable: true })
+  @Expose()
+  @IsOptional()
+  bmrProgramChart?: boolean
```

Comment:
Please add `@IsBoolean` under IsOptional and change `barProgramChart?: boolean` to `bmrProgramChart: boolean | null`